aries-site/src/pages/components/tabs.mdx

```diff
@@ -42,6 +43,25 @@ To maintain a clean, consistent presentation, all content should be aligned to t
 - Apply `justify="start"` to left-align the tabs and have their bottom border extend the full-width of the parent container.
 - Content within the Tab should be flush with the left edge of the Tabs component and have a vertical padding of `medium` to provide spacing and room for the content to breathe.
 
+### Responsive Tabs
+
+When horizontal space is limited, only the tabs that fit will be displayed. The remaining tabs can be accessed by using the previous and next arrows or with the keyboard.
+
+### Responsive Behavior
+
+- Previous and next arrows will be present if the horizontal tabs are fully outside the viewport.
+- Previous and next arrows are disabled if the horizontal tab has no more tabs to view towards the left or right.
+- Clicking on the previous and next arrows will show additional tabs. On a small screen the arrows will show 1 additional tab. On medium or large screens, clicking the arrows will show 3 additional tabs.
+
```

Comment:
It seems like we would not recommend having this many tabs, however we need a lot for demonstration purposes.

I can just see someone coming and pointing to this example as a Design System recommended best practice. That said, I'd like to think we wouldn't need to be McDonald's and put "CAUTION: Beverage is hot" on our coffee cups...

Could we cut down the number of tabs being displayed by using the `showResponsiveControls` prop which would allow the user to switch to a narrower view to see the effect? Alternatively, we could use the `width` prop to force the restraint. I'd tend to lean towards the former.

Reply:
I reduced the number of tabs to 6 (previously there were 9 tabs). I also added the `showResponsiveControls` prop so it can be viewed in mobile view which will show the responsive tabs behavior.